internal/handlers/common/aggregations/stages/projection/projection.go

```diff
@@ -577,31 +577,50 @@ func setBySourceOrder(key string, val any, source, projected *types.Document) {
 // processOperatorError takes internal error related to operator evaluation and
 // returns proper CommandError that can be returned by $process aggregation stage.
 func processOperatorError(err error) error {
```

Comment:
Could you fix go doc here too please? We don't have `$process` aggregation stage.

Reply:
Also could you add list of errors like https://github.com/FerretDB/FerretDB/blob/main/internal/handlers/common/aggregations/stages/projection/projection.go#L49

I think it also wise to add top level exported function `ProjectDocument`

Reply:
How about documenting returned error on exported function `ProjectDocument`?